scio-core/src/main/scala/com/spotify/scio/util/ClosureCleaner.scala

```diff
@@ -18,232 +18,19 @@
 package com.spotify.scio.util
 
 import java.io.NotSerializableException
-import java.lang.reflect.{Constructor, Field}
 
 import org.apache.beam.sdk.util.SerializableUtils
-import org.objectweb.asm.{ClassReader, ClassVisitor, MethodVisitor, Type}
-import org.objectweb.asm.Opcodes._
-
-import scala.annotation.tailrec
-import scala.collection.mutable.{Map => MMap, Set => MSet, Stack => MStack}
-import scala.language.existentials
-import scala.util.Try
 
 private[scio] object ClosureCleaner {
-
-  /** Clean the closure in place. */
-  def apply[T <: AnyRef](func: T): T = {
+  def apply[T <: AnyRef](obj: T): T = {
     try {
-      SerializableUtils.serializeToByteArray(func.asInstanceOf[Serializable])
+      SerializableUtils.serializeToByteArray(obj.asInstanceOf[Serializable])
     } catch {
       case e: IllegalArgumentException if e.getCause.isInstanceOf[NotSerializableException] =>
-        clean(func)
+        com.twitter.chill.ClosureCleaner(obj)
```

Comment:
Do we need this still? With `-Ydelambdafy:inline` I'm guessing this path is not exercised at all (and if it were, it could break - at best it will do nothing because `$OUTER` is no longer a thing, at worst it will crash).